kolibri/core/assets/src/navigation.vue

```diff
@@ -1,33 +1,12 @@
 <template>
-
   <header role="banner">
     <nav class="titlebar" role="navigation" aria-label="Main navigation">
```

Comment:
I'm hoping that this will be the only nav we need, simplifying the accessibility and the naming schema for navigation content.